source/reference/methods.txt

```diff
@@ -691,6 +747,88 @@ standalone and replica set deployments.
 
      - Returns the free cloud monitoring status for your deployment.
 
+Client-Side Field Level Encryption Methods
+------------------------------------------
+
+.. note:: Limitations
+
+   Automatic encryption is only available when ``mongosh`` is connected
+   to an Atlas cluster or a MongoDB Enterprise Server. For details, see
+   :manual:`Automatic Client-Side Field Level Encryption
+   </core/security-automatic-client-side-encryption/>`.
```

Comment:
It might be worth noting that all the methods here still work for non-enterprise servers, because they only perform manual encryption/decryption.

Reply:
Good point. I could see that being a point of confusion for readers.